draft-ietf-quic-tls.md

```diff
@@ -1552,30 +1561,39 @@ number of attempts to forge packets. TLS achieves this by closing connections
 after any record fails an authentication check. In comparison, QUIC ignores any
 packet that cannot be authenticated, allowing multiple forgery attempts.
 
-Endpoints MUST count the number of received packets that fail authentication for
-each set of keys.  If the number of packets that fail authentication with the
-same key exceeds a limit that is specific to the AEAD in use, the endpoint MUST
-stop using those keys.  Endpoints MUST initiate a key update before reaching
-this limit.  If a key update is not possible, the endpoint MUST immediately
-close the connection.  Applying a limit reduces the probability that an attacker
-is able to successfully forge a packet; see {{AEBounds}} and {{ROBUST}}.
+Endpoints MUST count the number of received packets that fail authentication
+for each set of keys and during the lifetime of a connection. If the number of
+packets that fail authentication with the same key exceeds a limit that is
+specific to the AEAD in use, the endpoint MUST stop using those keys.
+Endpoints MUST initiate a key update before reaching this limit.
+If a key update is not possible, the endpoint MUST immediately close the
+connection. If the number of packets that fail authentication within the
+connection (across all keys) exceeds a limit that is specific to the AEAD in
+use, the endpoint MUST immediately close the connection. Applying these limits
```

Comment:
An immediate close can still involve sending and receiving packets, so you need to add " and not attempt to process any further packets".  Or something along these lines.